.github/workflows/publish-abac-nuget.yml

```diff
@@ -0,0 +1,36 @@
+name: Pack and publish Altinn.Authorization.ABAC nuget package
+
+on:
+  pull_request:
+    branches: infrastructure/publish-nuget-action
+  release:
+    types:
+      - published
+
+jobs:
+  build-pack:
+    #if: startsWith(github.ref, 'refs/tags/Altinn.Authorization.ABAC-')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: |
+            2.0.x
+      - name: Install deps
+        run: |
+          cd src/Altinn.Authorization.ABAC
+          dotnet restore
+      - name: Build
+        run: |
+          cd src/Altinn.Authorization.ABAC
+          dotnet build --configuration Release --no-restore -p:Deterministic=true -p:BuildNumber=${{ github.run_number }}
```

Comment:
You should not need to set these values. Rather, the following two private dependencies should be added:

https://github.com/Altinn/altinn-authorization-utils/blob/a1deeb53af9573579aee66e0f6710691f2ac4c3f/Directory.Build.targets#L8

Reply:
i tested by adding the

Reply:
Symbols does not have anything to do with build-number.

Reply:
Exactly, but im not able to understand why that error goes away when i set the build number

Reply:
Ah. That we should be able to figure out quite easily by just building locally (or producing a binlog in the github action).